src/manufacturer/manufacturer.component.test.tsx

```diff
@@ -82,4 +82,24 @@ describe('Manufacturer', () => {
       ).toHaveStyle('background-color: inherit');
     });
   });
+
+  it('opens delete dialog and closes it correctly', async () => {
+    createView();
+    await waitFor(() => {
+      expect(screen.getByText('Manufacturer A')).toBeInTheDocument();
+    });
+
+    await user.click(
+      screen.getByRole('button', { name: 'Delete Manufacturer A manufacturer' })
+    );
+
+    expect(screen.getByText('Delete Manufacturer')).toBeInTheDocument();
+
+    const closeButton = screen.getByRole('button', { name: 'Cancel' });
+    await user.click(closeButton);
+
+    await waitFor(() => {
+      expect(screen.getByText('Actions')).toBeInTheDocument();
+    });
```

Comment:
Could this instead be something like
```suggestion
    await waitFor(() => {
      expect(
        screen.queryByText('Delete Manufacturer')
      ).not.toBeInTheDocument();
```
The 'Actions' heading will be in the document regardless of whether it closes or not.